core/services/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -440,9 +449,9 @@ func CreateEthTransaction(db *gorm.DB, fromAddress, toAddress common.Address, pa
 	// This is because they are not ever deleted if attached to an eth_tx that
 	// is moved into confirmed/fatal_error state
 	res := db.Raw(`
-INSERT INTO eth_txes (from_address, to_address, encoded_payload, value, gas_limit, state, created_at)
+INSERT INTO eth_txes (from_address, to_address, encoded_payload, value, gas_limit, state, created_at, meta)
```

Comment:
When we remove the v1 pipeline we can remove this meta field and have a real fk to the job run/task. Its like this right now to support both v1 and v2